core-services/prow/02_config/generate-boskos.py

```diff
@@ -99,7 +99,6 @@
     'azure-arm64-quota-slice': {
         'centralus': 3,
         'eastus': 3,
-        'eastus2': 3,
         'westus2': 3
```

Comment:
just thinking that perhaps we could change the zone to avoid quota issues while we run the tests or distribute these 3 slices to the other zones.

Reply:
right i had the same thought. which other zone supports arm64 instances that we can use?

Reply:
the `southcentralus` and `northeurope` regions support arm64 too, you can run `az vm list-usage --location southcentralus --output table | grep -E 'Name|Standard DPSv5 Family|Total Regional vCPUs' | sort;` to see which region has enough vCPU in your az subscription, a default cluster installation need 44 vCPUs.

Reply:
added southcentralus